docs/src/main/asciidoc/security-enabling-basic-auth-tutorial.adoc

```diff
@@ -0,0 +1,24 @@
+[id="security-enabling-basic-auth-tutorial"]
+= Enabling Basic Authentication
+
+The following tutorial describes how to enable the Basic Authentication for your Quarkus project and allow users to authenticate with a username and password.
+ 
+.Prerequisites
+ 
+* You have installed at least one extension that provides an `IdentityProvider` based on username and password, such as xref:security-jdbc.adoc[Elytron JDBC].
+ 
+.Procedure
```

Comment:
The how to _is_ a procedure. The template does include how/where to start numbered sections so the steps are obvious.

Reply:
Addressed in Michal's PR. Changed this doc type to how-to.